packages/ckeditor5-bookmark/tests/bookmarkediting.js

```diff
@@ -50,4 +59,90 @@ describe( 'BookmarkEditing', () => {
 			expect( model.schema.checkChild( [ 'paragraph' ], 'bookmark' ) ).to.be.true;
 		} );
 	} );
+
+	describe( 'dataDowncast', () => {
```

Comment:
I'd add more tests, for example with text before, after and surrounded with text.

Reply:
Addressed in 8211782.